frr-10.1.yaml

```
@@ -0,0 +1,111 @@
+package:
+  name: frr-10.1
+  version: 10.1.1
+  epoch: 0
+  description: The FRRouting Protocol Suite
+  copyright:
+    - license: GPL-2.0-only
+  dependencies:
+    provides:
+      - frr=${{package.full-version}}
+    runtime:
+      - python-3.11
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - bison
+      - build-base
+      - busybox
+      - elfutils-dev
+      - flex
+      - json-c-dev
+      - libcap-dev
+      - libtool
+      - libyang-dev
+      - linenoise-dev
+      - lua5.3
+      - lua5.3-dev
```

```suggestion
      - lua5.3
      - lua5.3-dev
```

These aren't needed as far as I can tell? I don't see anything lua related in the frrrouting/frr image

---

lua is very much required else the ./configure will not pass (ref: https://github.com/FRRouting/frr/pull/13239/files) this is required because upstream passed `--enable-scripting` to `configure` and I want to do the same so pulled in these lua deps.